war/src/main/webapp/help/system-config/homeDirectory.html

```diff
@@ -1,17 +1,22 @@
 <div>
-  Jenkins stores all the data files inside this directory in the file system.
-  You can change this by either:
-  <ol>
+  Jenkins stores all of its data in this directory on the file system.
```

Comment:
Yeah, I should have re-read this after updating the build and workspace directory help files!